Automattic/wp-calypso
# Stats: Remove the GA nudge on Countries summary page

Related to #

## Proposed Changes

Removes the GA nudge shown here:

<img width="736" alt="SCR-20241218-omhf" src="https://github.com/user-attachments/assets/d03c38b2-b04c-4015-a0c8-4e9f0731abc6" />

Did a quick search for other instances of "Google Analytics" and the `UpsellNudge` component and did not find any other instances that might need removing.

## Why are these changes being made?


p1734094110750839-slack-C068X5XJT3J

## Testing Instructions



* Visit the Traffic page on a test site without a plan but with access to the Countries module.
* Click the "View all/more" link on the Countries module.
* Confirm the upgrade nudge is no longer shown.

## Pre-merge Checklist
